grafana/scylla-advanced.template.json

```diff
@@ -27,6 +27,81 @@
                 ],
                 "title": "New row"
             },
+            {
+                "class": "row",
+                "height": "25px",
+                "panels": [
+                    {
+                        "class": "percentunit_panel",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "sum(rate(scylla_io_queue_consumption{class=~\"$classes\", instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\"}[1m])) by ([[by]], class)",
```

Comment:
I didn't understand the comment, this is an example of the metric:
```
# TYPE scylla_io_queue_consumption counter
scylla_io_queue_consumption{class="commitlog",ioshard="0",mountpoint="none",shard="0",stream="rw"} 0.015914
```

What label are you referting to?

Reply:
What version of scylla is it?

Reply:
E.g. current master is
```
# TYPE scylla_io_queue_consumption counter
scylla_io_queue_consumption{class="commitlog",iogroup="0",mountpoint="none",shard="0",stream="rw"} 0.007846
```

and scylla-5.4.0-rc1 (should) have the same (according to the seastar code)

Reply:
This is from Enterprise 2023.1.2

Reply:
Should we have graphs per class per iogroup? (cartesian product)

Reply:
@xemul ping, please see my question

Reply:
Sorry

> Should we have graphs per class per iogroup? (cartesian product)

Yes, stacked by class

> This is from Enterprise 2023.1.2

I see. https://github.com/scylladb/seastar/pull/1591 is not yet there. Since it's not a bugfix, I don't think it will be backported. Well, for versions without iogroup label we'll have some inconveniences then, but it's possible to live with them